crates/api/src/site/registration_applications/tests.rs

```diff
@@ -34,7 +34,7 @@ use lemmy_db_views::structs::LocalUserView;
 use lemmy_utils::{error::LemmyResult, LemmyErrorType, CACHE_DURATION_API};
 use serial_test::serial;
 
-#[allow(clippy::unwrap_used)]
+#[expect(clippy::unwrap_used)]
```

Comment:
This is fine, but eventually I'd like to remove all these `unwrap_used` lints (and replacing it with `fn test___() -> LemmyResult<()>` . 

I've been doing it as I see them, but I'll make an issue for it.